pip/cmdoptions.py

```diff
@@ -523,6 +523,56 @@ def only_binary():
 )
 
 
+def strong_hashes():
```

Comment:
Just on the chance that someone treats it as a public API and hashlib grows a way to dynamically determine hash strength. So: only a very slight one.